.circleci/config.yml

```diff
@@ -124,6 +124,18 @@ jobs:
               "${CIRCLE_TAG}" \
               csv2rdf-darwin.tar.gz
 
+  deploy-docker:
+    docker:
+      - image: cimg/clojure:1.11.1
```

Comment:
I personally prefer locking the image to a known JDK version which we intend to support e.g. this snippet from the muttnik project:


<img width="506" alt="Screenshot 2023-08-08 at 3 16 47 PM" src="https://github.com/Swirrl/csv2rdf/assets/241747/384490bc-a486-48bf-b1d1-0d6ca520b898">